### DISCUSSION TITLE
No data in table "tracking_summary" for ckan 2.9.5

### DISCUSSION THREAD

**Discussion by muhammetcetinyamac** [ckan/ckan#7048] [Q&A] [answered]
Hi, I'm using ckan 2.9.5. Unfortunately, there is no data entry into the `tracking_summary` table. Could I have made a mistake in the installation? Or has this feature been removed?
Thank you for your interest.

**Comment by jqnatividad** [comment, marked as answer]
Hi @muhammetcetinyamac ,

You need to ensure that page view tracking is fully configured and the cron job that updates the `tracking_summary` table is running.

https://docs.ckan.org/en/2.9/maintaining/tracking.html#enabling-page-view-tracking

**Reply by muhammetcetinyamac** [reply]
Thank you. I will review the document.

**Reply by muhammetcetinyamac** [reply]
@jqnatividad  Thank you very much again. I solved my problem by reviewing the document.

**Reply by jqnatividad** [reply]
No problem @muhammetcetinyamac.
Please select the answer so other folks with the same query can quickly find it.
